CoolBook/Views/Users/Index.cshtml

```diff
@@ -23,36 +23,59 @@
     <thead>
         <tr>
             <th>
-                @Html.DisplayNameFor(model => model.UserName)
+                @Html.DisplayNameFor(model => model.User.UserName)
             </th>
             <th>
-                @Html.DisplayNameFor(model => model.Email)
+                @Html.DisplayNameFor(model => model.User.Email)
             </th>
             <th>
-                @Html.DisplayNameFor(model => model.Role)
+                @Html.DisplayNameFor(model => model.User.Role)
```

Comment:
Why did you add these to the index?
The profile is where users can interact with this

Reply:
In order to use JOIN, I needed a place without UserInfo, and considering the search option (which uses these parameters...)